src/pages/Scholarships.jsx

```diff
@@ -53,17 +71,16 @@ function ScholarshipsPage() {
   const sortBy = params.sortBy ?? DEFAULT_SORT_BY;
 
   if (
-    sortBy !== 'deadlineAsc' &&
+    sortBy !== DEFAULT_SORT_BY &&
     sortBy !== 'deadlineDesc' &&
     sortBy !== 'amountAsc' &&
     sortBy !== 'amountDesc'
   ) {
     pruneQueryParam('sortBy');
   }
 
-  const key = params.sortBy !== undefined ? sortBy : DEFAULT_SORT_BY;
-  const sortField = sortParams[key].field;
-  const sortDir = sortParams[key].dir;
+  const sortField = sortOptions[params.sortBy ?? DEFAULT_SORT_BY].field;
+  const sortDir = sortOptions[params.sortBy ?? DEFAULT_SORT_BY].dir;
```

Comment:
Again, `sortBy` is already equal to `params.sortBy ?? DEFAULT_SORT_BY`. There's no need to repeat the expression.

Reply:
Ok